src/web/app/views/beambox/Right-Panels/ActionsPanel.tsx

```diff
@@ -212,16 +218,43 @@ class ActionsPanel extends React.Component<Props> {
         <ArrayIcon />,
         <ObjectPanelIcons.Array />
       ),
-      this.renderButtons(
+      remove: this.renderButtons(
         'Potrace',
         () => imageEdit.potrace(elem as SVGImageElement),
         false,
         'remove',
         false,
-        <ActionPanelIcons.Invert className={styles.icon} />
+        <TraceIcon />,
+        <ObjectPanelIcons.Trace />
```

Comment:
new trace and potrace icon had been added in the develop branch, please check https://github.com/flux3dp/beam-studio-core/commit/62ac9d04d1be163598253c2cd39e2352a65a5a0d